client/hwctl/src/main.rs

```diff
@@ -27,7 +27,16 @@ use hwlib::{
 
 #[tokio::main]
 async fn main() -> Result<()> {
-    let cert_status_request = CertificationStatusRequest::new(Paths::default())?;
+    let cert_status_request = match CertificationStatusRequest::new(Paths::default()) {
+        Ok(request_data) => request_data,
+        Err(e) => {
+            eprintln!("failed to collect system data: {}", e);
```

Comment:
Just to follow [the convention](https://github.com/canonical/rust-best-practices/blob/main/best-practices.md#error-messages), I think this error message should start with the word 'cannot' rather than 'failed to.' Also, I think it would be nice to inline that format parameter, as in `"... {e}"`